### PR TITLE
adding cheek teeth

### PR DESCRIPTION
**Pull request by megbalk**
issue #2585

### CODE REVIEW

File: src/ontology/uberon-edit.obo [hunk 1]
```diff
@@ -222488,6 +222488,16 @@ relationship: dc-contributor http://orcid.org/0000-0003-2699-3066
...
 
+[Term]
+id: UBERON:7500108
+name: cheek teeth
+def: "The post-canine teeth (molars and premolars) that occur at the zygomatic plate." [Wikipedia:Cheek_teeth]
+synonym: "cheekteeth" EXACT []
+is_a: UBERON:0013164 ! molariform tooth
+relationship: dc-contributor https://orcid.org/0000-0001-8415-4848
+relationship: dc-contributor https://orcid.org/0000-0003-2699-3066
+relationship: part_of UBERON:0013217 ! zygomatic plate
```
Comment (cmungall) [inline src/ontology/uberon-edit.obo:222499]: the zygomatic plate is bony, is the cheek tooth really part of the bone?

compare with

```yaml
id: UBERON:0003267
name: tooth of upper jaw
def: "A calcareous tooth that is part of the upper jaw region. In mammals, the upper jaw teeth are attached to the maxilla." [http://orcid.org/0000-0002-6601-2165]
synonym: "maxillary tooth" NARROW [NCBITaxon:40674]
synonym: "upper jaw tooth" EXACT [MA:0001909]
xref: EHDAA:8045
xref: EMAPA:17938
xref: MA:0001909
xref: NCIT:C49792
xref: SCTID:422279005
xref: UMLS:C1710581 {source="ncithesaurus:Upper_Jaw_Tooth"}
intersection_of: UBERON:0001091 ! calcareous tooth
intersection_of: part_of UBERON:0001709 ! upper jaw region
relationship: attaches_to UBERON:0011597 ! bone of upper jaw
relationship: part_of UBERON:0009680 ! set of upper jaw teeth
property_value: editor_note "Note that we follow EHDAA2 and consider teeth part of the jaw skeleton" xsd:string
```

File: src/ontology/uberon-edit.obo [hunk 2]
```diff
@@ -222488,6 +222488,16 @@ relationship: dc-contributor http://orcid.org/0000-0003-2699-3066
 relationship: part_of UBERON:7500094 ! tubercle of calcaneus
 property_value: dcterms-date 2022-05-19T21:16:42Z xsd:dateTime
 
+[Term]
+id: UBERON:7500108
+name: cheek teeth
```
Comment (cmungall) [inline src/ontology/uberon-edit.obo:222493]: use singular

File: src/ontology/uberon-edit.obo [hunk 3]
```diff
@@ -222488,6 +222488,16 @@ relationship: dc-contributor http://orcid.org/0000-0003-2699-3066
...
+def: "The post-canine teeth (molars and premolars) that occur at the zygomatic plate." [Wikipedia:Cheek_teeth]
+synonym: "cheekteeth" EXACT []
```
Comment (cmungall) [inline src/ontology/uberon-edit.obo:222495]: add synonym type PLURAL

**Review by bvarner-ebi — CHANGES_REQUESTED** [review]
Hi @megbalk, I concur with @cmungall's comments: 
- singular form of the term should be used in the label and definition, per convention
- the `relationship: part_of UBERON:0013217 ! zygomatic plate` axiom is a problem for the reason pointed out by @cmungall and also because it is not applicable to the lower cheek teeth.

File: src/ontology/uberon-edit.obo (hunk 1 above)
Comment (megbalk) [reply src/ontology/uberon-edit.obo:222499]: I realized that I was looking at upper cheek teeth, but cheek teeth are both upper and lower, so I will change.

**Comment by megbalk** [comment]
@bvarner-ebi by singular form do you mean cheek tooth?

**Comment by bvarner-ebi** [comment]
> @bvarner-ebi by singular form do you mean cheek tooth?

Hi @megbalk, yes. 
Conventionally singular forms are used in labels and definitions.

**Comment by megbalk** [comment]
should there be a term "cheek tooth" referring to any post-canine teeth and then a term "cheek teeth" for the grouping of post-canine teeth?

**Comment by megbalk** [comment]
I may need to add cheektooth row, under "tooth row" to refer to the set of cheek teeth. Should I make a new issue for that or add it to the current one?

**Comment by bvarner-ebi** [comment]
> I may need to add cheektooth row, under "tooth row" to refer to the set of cheek teeth. Should I make a new issue for that or add it to the current one?

At this stage it may be easier to finalise this issue and extend the additional work in a separate issue that references the current issue.

**Comment by megbalk** [comment]
Great, I can do that. I made it singular. Was that the only change?

**Comment by bvarner-ebi** [comment]
> Great, I can do that. I made it singular. Was that the only change?

I think @cmungall is suggesting to use the synonym type plural, similar to how it is used for UBERON:0004346 'gubernaculum (male or female)', as an example.

Screenshot 2022-08-19 at 09 37 39

I'm not familiar with the use cases for the property type 'plural term', which may be OK, but I do not see that used elsewhere in the ontology.

File: src/ontology/uberon-edit.obo [hunk 4]
```diff
@@ -222488,6 +222488,17 @@ relationship: dc-contributor http://orcid.org/0000-0003-2699-3066
 relationship: part_of UBERON:7500094 ! tubercle of calcaneus
 property_value: dcterms-date 2022-05-19T21:16:42Z xsd:dateTime
 
+[Term]
+id: UBERON:7500108
+name: cheek tooth
+def: "Any of the post-canine teeth, which are the multi-cusped molars or premolars used for shearing and grinding food in mammals." [Wikipedia:Cheek_teeth]
+synonym: "cheek teeth" RELATED [] {synonymtypedef="PLURAL"}
```
Comment (bvarner-ebi) [inline src/ontology/uberon-edit.obo:222495]: I think the refs:label is 'plural term'.
See UBERON:0004865 in Protégé as an example.

File: src/ontology/uberon-edit.obo [hunk 5]
```diff
@@ -222488,6 +222488,17 @@ relationship: dc-contributor http://orcid.org/0000-0003-2699-3066
 relationship: part_of UBERON:7500094 ! tubercle of calcaneus
 property_value: dcterms-date 2022-05-19T21:16:42Z xsd:dateTime
 
+[Term]
+id: UBERON:7500108
+name: cheek tooth
+def: "Any of the post-canine teeth, which are the multi-cusped molars or premolars used for shearing and grinding food in mammals." [Wikipedia:Cheek_teeth]
```
Comment (bvarner-ebi) [inline src/ontology/uberon-edit.obo:222494]: Re: "multi-cusped"
Consider replacing with "multicuspidate" or "multi-cusp".

**Review by bvarner-ebi — APPROVED** [review]
Approved.
Consider word change in definition as noted.

**Pull request merged**